htdocs/core/modules/printing/modules_printing.php

```diff
@@ -69,7 +69,7 @@ public static function listDrivers($db, $maxfilenamelength = 0)
 		$listoffiles = array();
 		$dirmodels = array_merge(array('/core/modules/printing/'), (array) $conf->modules_parts['printing']);
 		foreach ($dirmodels as $dir) {
-			$tmpfiles = dol_dir_list(dol_buildpath($dir, 0), 'all', 0, '\modules.php', '', 'name', SORT_ASC, 0);
+			$tmpfiles = dol_dir_list(dol_buildpath($dir, 0), 'all', 0, '.modules.php', '', 'name', SORT_ASC, 0);
```

Comment:
I think it should be
'\/modules.php'
don't you think so ?

Reply:
we need to find `/core/modules/printing/`abcdef`.modules.php`